### PR TITLE
feat(23.10):add archive for curl in mantic

### PR DESCRIPTION
**Pull request by yashashwi**
**# Proposed changes**
**Adding archive for CURL and its transitive dependencies**

This PR is same as https://github.com/canonical/chisel-releases/pull/127 but with CLA approved.
This change is already available in 22.04 and 24.04

### CODE REVIEW

**Review by zhijie-yang — APPROVED** [review]
Looks good to me.

File: slices/libldap2.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - libc6_libs
+      - libgnutls30_libs
+      - libsasl2-2_libs
+    contents:
+      /usr/lib/*-linux-*/liblber.so.2*:
+      /usr/lib/*-linux-*/libldap.so.2*:
+
+  copyright:
+    contents:
+      /usr/share/doc/libhogweed6/copyright:
```
Comment (linostar) [inline slices/libldap2.yaml:18]: This should be `/usr/share/doc/libldap2/copyright:`.

**Review by linostar — CHANGES_REQUESTED** [review]

File: slices/libldap2.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply slices/libldap2.yaml]: changed

**Review by cjdcordeiro — APPROVED** [review]
ty. will wait for @rebornplusplus 's review and then merge

**Review by linostar — APPROVED** [review]
LGTM, thanks!

**Pull request merged**